### DISCUSSION TITLE
how to use live collaboration with my organization server

### DISCUSSION THREAD

**Discussion by Sohailamjad1** [excalidraw/excalidraw#4685] [Q&A]
Hi ..can you please help me for implementing the live collaboration with my organization server

**Comment by Sohailamjad1** [comment]
where i can add my organization people and i can share the link with them

**Comment by ad1992** [comment]
Its already answered [here](https://github.com/excalidraw/excalidraw/discussions/3879#discussioncomment-1110524) @Sohailamjad1

**Reply by Sohailamjad1** [reply]
hi @ad1992 ..it is little bit tricky to understand so can you send the video clip or exact code with sandbox example...and one more question all live collaboration part is handle from front end side or backend side also need??

**Comment by Sohailamjad1** [comment]
Hi @ad1992 ...can you pls help

**Reply by ad1992** [reply]
@Sohailamjad1 as mentioned in the above answer you can take a look at the [collaboration wrapper](https://github.com/excalidraw/excalidraw/blob/master/src/excalidraw-app/collab/CollabWrapper.tsx) as in the excalidraw app we are using the npm package and have implemented the collaboration so you can as well copy the same code to support collaboration

**Reply by Sohailamjad1** [reply]
all the things in react? and backend api also need?

**Reply by Sohailamjad1** [reply]
Hi can you tell me npm package name pls?

**Reply by Sohailamjad1** [reply]
> Hi can you tell me npm package name pls?
??

**Reply by ad1992** [reply]
Its the same npm package [@excalidraw/excalidraw ](https://www.npmjs.com/package/@excalidraw/excalidraw), since the code base for excalidraw app and excalidraw package is same so we are directly importing `excalidraw` from the code base instead of the packge, the entry point of the package is [this file](https://github.com/excalidraw/excalidraw/blob/master/src/excalidraw-app/index.tsx), you will notice we are importing `Excalidraw` in this file

**Reply by Sohailamjad1** [reply]
ok so, no need to install any packages , so do i need to create one more collab file and paste it all the collab code which you have given in this this link  [collaboration wrapper](https://github.com/excalidraw/excalidraw/blob/master/src/excalidraw-app/collab/CollabWrapper.tsx)
and do i also need to create one index.tsx file ...for excalidraw iam using only four file ngxexcalidraw.tsx, reactWrapper.tsx and reactModule and initial data.ts file
